### ISSUE TITLE
Prometheus metrics customization

### ISSUE THREAD

**Issue by sorobon** [idealista/nginx_role#19]
Currently ansible role always install and configure prometheus metrics under '/metrics' path.
We need:
- Not install prometheus metrics using a flag or something similar.
- Change the path used to expose metrics using a variable.

It may look like this:

```yaml
## Metrics
prometheus_metric_enabled: true
prometheus_metric_port: 9145
prometheus_metric_role: nginx
prometheus_metric_path: /metrics
```

**Comment by dortegau** [comment]
It seems reasonable to pull apart all the metrics tasks in other file (¿metrics.yml?) 👍

**Comment by sorobon** [comment]
Merged in release 1.3.0 :)
